galleries/examples/mplot3d/view_planes_3d.py

```diff
@@ -16,13 +16,13 @@ def annotate_axes(ax, text, fontsize=18):
     ax.text(x=0.5, y=0.5, z=0.5, s=text,
             va="center", ha="center", fontsize=fontsize, color="black")
 
-# (plane, (elev, azim, roll))
```

Comment:
This change removes the ability to copy-paste these docs into ones functions, and is confusing wrt the keyword ordering below.